tooltester/src/test/java/io/dockstore/tooltester/client/cli/ClientTest.java

```diff
@@ -119,8 +118,7 @@ public void enqueueUnverifiedTool() {
      */
     @Test
     public void createJenkinsTests() {
-        String[] argv = { "sync", "--execution", "jenkins", "--api",
-                "https://www.dockstore.org:8443/api/ga4gh/v1" };
+        String[] argv = { "sync", "--execution", "jenkins", "--api", "https://www.dockstore.org:8443/api/ga4gh/v1" };
```

Comment:
I believe the /.dockstore/config file's contents don't really matter since test launches are using local files.  The config file is there so that dockstore doesn't complain.